src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -193,34 +259,57 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
       // Only message self to perform the transition after the persistence of the new state is complete.
       transitionFuture map { _ => PerformTransition(toState) } pipeTo self
     }
+  }
 
-    stay()
+  /**
+   * Captures the `ExecutionStoreKey`/`ExecutionStatus` combination for a call that has been created or started
+   * as part of `startRunnableCalls`.  These are tracked to coordinate processing of messages based on completion
+   * of persistence.
+   */
+  case class StartEntry(execution: ExecutionStoreKey, status: ExecutionStatus)
+
+  /**
+   * Collects the `StartEntry`s which have been persisted as part of starting runnable calls, as well as
+   * any new entries which were added to the execution store (exploding scatters).
+   */
+  case class ExecutionStartResult(startEntries: Traversable[StartEntry],
+                                  newEntries: Traversable[ExecutionStoreKey] = Seq.empty) {
+    /** Merge two `ExecutionStartResult`s. */
+    def +(other: ExecutionStartResult): ExecutionStartResult = {
```

Comment:
This might be the first one I've done in Cromwell.  I can give it a normal boring function name if you like, but I thought the '+' concisely conveyed that this was operating on two arguments to return a new value, as opposed to mutating anything in place.